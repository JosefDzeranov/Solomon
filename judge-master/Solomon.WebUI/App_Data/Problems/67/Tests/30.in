1000
122173734 167625006 119892
904734407 17588418 948792
725976448 113009874 641229
283266743 26370707 526741
630858285 112504076 31545
20245186 58611717 488905
160932458 820502332 97757
80623029 566035421 550637
306008073 373267328 241737
804555630 177692948 514692
261493515 41651603 766107
704898917 54749690 848456
636686676 17621263 246658
802809458 29494395 521901
832303853 97319166 71094
289798740 508012980 940478
315836198 389062719 115579
24991251 2489240 843608
194106305 358850804 78836
505474165 333706706 269225
552783624 406245171 602591
162560631 412628417 648119
548666466 264473246 107216
68762872 234382246 71560
250554077 512545400 386437
27480491 93678556 743026
194106305 111673187 734574
473666663 49969695 903323
47117715 851460266 370472
385785510 166998114 776190
135388551 763189430 428950
199542793 92612060 449456
85592555 451935864 666460
24991251 600807836 425243
456764523 143408175 956242
261493515 48143935 201819
665868549 69315142 306887
250554077 273082281 895734
802809458 179439120 112786
283266743 158094299 120972
29946031 833199667 722678
78104599 504235039 20748
124598293 137726018 798831
29172630 39590242 440284
224251850 412434826 977817
654008304 51659384 409348
250035072 33231671 287065
20245186 894504742 563765
328282671 342719792 469960
630858285 155698394 723329
309637450 104022614 205814
292154853 626221771 184062
47117715 806233101 897002
292154853 512400777 121112
708177558 260646095 44764
381125300 516534248 316023
359619383 444936247 418012
378330831 424478627 385522
441361042 477015582 100076
413660064 210170303 330081
31158977 162947328 887777
735183691 77956021 463618
185286207 248275359 142127
592808611 354600847 303088
283266743 514544977 339037
433561566 358068278 993187
20245186 407267632 462464
292154853 570361421 355267
646658450 144971394 677333
249808348 140085852 761240
763099477 90251339 30133
292154853 88970447 624227
359619383 26166127 706199
625799087 39831733 119232
378330831 303147012 697052
302575811 234952608 340642
734130555 1053136 678841
681477843 45912079 878182
433561566 548687012 570581
725976448 255458342 873194
600842539 35844137 535924
224251850 455023551 288457
725976448 221433010 438335
121159047 274117505 431420
305779492 231748927 831739
162560631 153275567 354709
318003297 234780327 174667
185286207 311879645 674643
262324311 151335753 71338
378330831 347645617 80714
135388551 569510366 547383
65113360 128992945 244817
359619383 375564308 85063
65113360 240666132 554716
395276552 157525874 866820
162560631 699955643 249535
239849681 204647173 713583
70175164 837845274 41893
441361042 302001319 711410
29172630 878847808 704272
763099477 75886845 437235
280780628 320061911 443898
552957109 311933232 564351
31158977 207713974 107064
280780628 320061911 67927
654008304 11622516 731615
250035072 180679586 420100
52047090 329078210 520239
384792609 241006478 520192
292154853 231481505 184843
50066413 712065167 161878
239849681 265080647 297287
705667688 4383809 172923
430714658 408271664 19625
497165852 212885645 227658
238872951 142252349 50648
413660064 186512634 53630
239849681 84136665 521136
518018405 407603602 964647
444496854 155675844 351719
430714658 144474390 238177
588185697 203444147 976878
293106028 568076655 256051
199542793 533167378 532238
24991251 718371110 669821
625799087 303823932 212796
47117715 810208157 609495
283266743 570084073 828640
250554077 135231433 290086
289798740 584582141 270317
224251850 324414616 407623
588185697 91089704 440043
864890341 131872053 241536
78856903 244436946 905672
121159047 826250411 492128
261493515 581290107 630442
80623029 520219510 806682
763099477 41456153 8184
323986346 592605882 66367
504930328 149077976 210386
283721569 590659312 119844
804555630 142853828 411743
315836198 338172106 152235
249808348 5713880 957316
732710171 110073451 636755
575189048 12996649 390441
497165852 236964703 935505
80623029 758363293 532515
283266743 382364077 810492
654307939 16694524 247706
70175164 792970534 635366
65113360 640554328 881318
813139712 145889083 17898
389894200 446159778 231523
582339638 365069820 244646
315836198 553051962 729790
303145118 2862955 675199
50066413 785987565 739963
762131580 102758761 359264
427512818 251762583 187499
603612872 122363576 885594
29946031 22101059 612787
315836198 236966228 723338
85592555 341920263 537091
441361042 63569286 390744
255522228 264169538 775631
473666663 152132424 455569
250035072 485148619 650979
519691766 402631059 558048
600842539 238338332 81753
78856903 846765104 691226
283721569 573604303 845183
70175164 357337654 423749
162560631 468297654 927704
523636358 445187295 931806
160932458 94589770 116471
283266743 326656599 103909
303145118 249811991 629421
80623029 901625549 768811
519691766 62647872 266767
302575811 15427486 690071
404097065 543312393 282861
302575811 272613237 616742
385785510 612221181 767598
274659604 452730318 253131
255522228 282006191 122175
250554077 618334083 887600
224251850 209309716 532498
255522228 726726350 71954
162560631 742173776 66828
65113360 770940618 291470
262324311 499807269 916799
301040433 485516246 395867
29946031 220089041 277367
194106305 381082743 801858
592808611 169322969 961019
862516274 59806551 902562
725976448 172601533 982843
135388551 572789007 842778
250554077 267464328 717665
31158977 678892520 699269
381125300 142511058 720964
582339638 399095152 796823
665868549 97230928 168204
505474165 176003678 746754
305779492 80006018 652032
922322825 59111965 673445
283266743 639056082 674824
441361042 82275316 411705
274659604 557644249 30833
603612872 393149522 664102
552802426 260337286 84927
743362361 54449359 292180
473666663 157191622 630770
36080087 483611679 827365
78856903 846765104 387908
31158977 594640110 22109
47117715 77480578 217557
24991251 236502264 487122
194106305 514071253 847200
588185697 269140175 537241
681477843 44498605 584928
385785510 167016916 289244
315836198 294087144 411608
283266743 309541868 709991
289798740 692449838 16384
838986322 29901838 158270
504930328 222459594 279354
283721569 154172073 259774
518018405 301537805 521047
238872951 523258629 845306
289798740 632524085 232464
274552368 111233142 774225
323986346 199650012 29852
255522228 319666820 380579
705667688 253361107 947850
68762872 860860147 836941
819556210 139472585 950111
85592555 619306362 593827
582339638 127711859 726240
255522228 67771621 734122
24991251 611695425 843157
456764523 144078016 491206
600172698 81305145 229615
80623029 758557842 882990
121159047 1014687 607239
47117715 562805627 394541
85592555 832784069 114739
636686676 125444904 433139
600172698 325449309 388259
280780628 617797353 343521
293106028 144787614 137070
24991251 284646199 462868
318003297 409386625 310603
250035072 747971619 430045
24991251 512537168 488451
603612872 129097299 282460
391382486 421757226 277125
194106305 603705415 874669
441361042 394692936 735845
306008073 616314752 957651
378330831 329846727 529826
600172698 307847740 487362
24991251 656486592 867883
473666663 383659209 580483
36080087 589719000 352030
612072408 256815752 2816
121159047 352507616 47544
427512818 569249576 612516
239849681 84136665 408875
292154853 520984859 950998
708177558 145173258 417629
185286207 744336812 45546
262324311 403306509 1999
68762872 186759356 496446
444496854 481125153 991302
654008304 327426486 839407
735183691 125998992 151491
78856903 684242574 997922
665868549 256454276 474522
759648607 105241734 357786
727389922 177344485 491983
391382486 132253872 54238
456764523 209104026 734697
404097065 510652863 339589
378330831 247468256 850386
20245186 371137300 651166
121159047 584508641 384835
302575811 307347531 993368
708177558 35184803 857886
249808348 431669495 503977
31158977 284677221 820601
78104599 231532851 552979
384792609 168164500 611248
762131580 156245044 239313
665630820 232028728 943703
381125300 381974177 537194
24991251 893385373 778344
505474165 463349488 864539
250554077 254376251 771488
280780628 580402055 22293
441361042 350268802 962630
625799087 210254891 90326
914749928 32659530 306265
679275401 289548252 516667
80623029 682476448 934008
124598293 399038065 841231
630858285 95118163 135778
224251850 203260968 592071
504930328 120868759 860435
20245186 218627765 313621
763099477 155277147 866320
122173734 613009957 488620
381125300 167541166 936560
413660064 223026612 903253
20245186 410469472 56012
29172630 1986347 550960
654008304 271613703 775076
261493515 313695533 779911
582339638 43459449 625144
763099477 99416797 585073
122173734 351492929 494451
413660064 504716560 535311
708177558 124126295 66365
85592555 35566492 440535
274552368 362134308 368012
813139712 55748448 589852
52047090 485481329 633732
603612872 365210781 110944
303145118 249638506 518192
36080087 594778198 944228
904734407 16686272 237595
85592555 836730270 244793
135388551 790233456 462246
274552368 182212155 310300
24991251 494700515 718615
497165852 322390358 659415
588185697 37613390 473436
262324311 529305533 978648
250554077 667822547 499432
24991251 225043821 156664
162560631 484097819 28283
384792609 88874054 26813
121159047 39773411 839088
122173734 152378634 435696
124598293 136895222 357670
328282671 259903026 887241
65113360 319679249 380852
274552368 684476427 36001
238872951 373199457 299135
274659604 129437461 297239
359619383 505270958 459921
47117715 860902723 639786
239849681 617476191 307769
600842539 218713671 866739
31158977 831357297 176377
78104599 46493694 116247
52047090 929387700 738012
274659604 48634245 937214
323293849 55036982 149794
710051497 129129374 668404
65113360 640554328 517009
280780628 635811600 345591
199542793 241818249 792232
768413772 63890081 507581
289798740 442911431 339643
65113360 134429433 480372
609923342 226130636 779253
323986346 226173258 544566
29172630 624835674 714076
194106305 608703153 473450
671002463 34665225 356729
24991251 268114777 896411
262324311 347599031 326638
65113360 773872962 688389
518018405 64321233 778112
292154853 543899125 644570
600842539 64788281 76715
389894200 518126238 564622
395276552 109653776 439027
441361042 264306646 810395
274552368 564628503 903479
427512818 487237110 393601
292154853 300653758 889402
262324311 255694094 837881
255522228 19030140 303167
68762872 361951786 857321
768413772 84937044 889478
121159047 129395030 291939
359619383 621815407 462594
427512818 332135789 793009
898577981 9442457 33576
261493515 28305225 190174
679275401 112354443 334668
505474165 347876651 943011
433561566 334852206 183149
588185697 155176664 309856
630858285 160771559 750966
250035072 477354850 314703
78856903 740699307 853765
135388551 302505091 754864
603612872 377821918 399228
262324311 211342352 115970
194106305 209990760 152757
160932458 786477000 236746
918376624 7245383 431773
433561566 334852206 881013
224251850 483925708 589948
523636358 141994462 72139
797811720 127810287 347708
78104599 919902092 761066
768413772 153909053 867132
85592555 99693652 484028
305779492 107880572 338132
763099477 56456733 370645
389894200 467431672 767842
537528419 205833942 136129
552957109 308225574 156398
78856903 311037297 904728
283721569 478410011 117654
437893642 216414297 640853
280780628 163716226 412607
36080087 15967003 236514
135388551 476683857 663854
323986346 149680317 86746
359619383 566002624 711685
224251850 81527642 121598
359619383 177909036 272422
224251850 422406600 730711
681477843 123077787 215700
612072408 53796141 819393
27480491 893940188 489797
378330831 540045793 790259
194106305 669039393 301880
250035072 475941376 349143
437893642 404889980 785405
249808348 592975274 801141
135388551 703597771 89922
29172630 733926847 754431
863145698 119102880 375552
289798740 123861324 725502
318003297 257185751 394996
27480491 247179113 42940
29946031 895675976 958637
444048851 319050626 426032
473666663 236384834 298941
238872951 50925789 692278
548666466 290514405 598539
78856903 245129443 666061
625799087 272778894 701344
239849681 193711885 623394
199542793 350616811 66414
27480491 275664627 30350
65113360 721443319 64999
47117715 950888976 61422
623830367 357604423 881499
592808611 243245367 967011
121159047 622203314 681379
20245186 905376821 863835
592808611 72822209 291655
630858285 182281427 163449
280780628 24998864 149399
552957109 238672735 115359
389894200 23765864 385649
261493515 288666089 991686
315836198 320850478 142584
31158977 271416834 771688
708177558 239231900 492842
122173734 646240038 401520
630858285 104325406 229747
391382486 212230386 690437
302575811 571805070 519109
763099477 141634930 638142
27480491 880539947 573839
293106028 88019272 638549
708177558 206572370 671700
725976448 221433010 220993
504930328 98682544 201235
78856903 223718908 98595
665630820 67079351 214586
262324311 12228057 420147
378330831 569078627 721317
68762872 388001651 368982
24991251 557348387 930516
404097065 9562999 59855
85592555 35566492 610683
441361042 162251830 134419
78104599 307680911 112767
283721569 129938495 367557
504930328 47872098 592045
395276552 443709770 882274
274552368 642039860 923554
250554077 52021734 378969
665868549 232709432 576686
293106028 611628379 604046
552802426 182381265 114823
78104599 474679025 372236
261493515 655098713 38951
384792609 197547029 892598
315836198 221692221 839456
20245186 818935685 863308
857325872 3856811 791475
24991251 236502264 358117
224251850 698070975 439233
654008304 56043193 306949
679275401 302159389 151436
250035072 302767354 672455
853350816 9165458 466037
27480491 699909431 115365
802809458 29494395 849364
80623029 352938537 261803
552783624 101224680 631155
519691766 33091858 135395
523636358 51552690 768326
274659604 328953268 739248
29172630 35940730 56535
121159047 515527629 853619
303145118 554180754 850859
315836198 419347493 208017
315836198 221692221 966783
27480491 50624108 689767
504930328 174345073 386694
385785510 167171599 257283
612072408 42235531 76320
78856903 757197075 390437
29946031 220089041 378647
456764523 541242168 993132
306008073 359860476 98868
305779492 89497060 136767
292154853 331675514 289270
292154853 311458019 518416
609923342 288654639 634182
121159047 741357227 505582
305779492 675655298 178610
389894200 378519572 691075
444048851 538199727 624066
725976448 9207243 140997
185286207 524765290 662719
50066413 553546459 258067
600172698 381262092 227271
199542793 527847129 332054
224251850 441378970 898038
185286207 117289604 556371
261493515 697535280 546112
80623029 229014421 5090
708177558 160710602 628082
623830367 238685907 62416
199542793 527847129 451445
24991251 900630756 665735
381125300 548497719 373830
283721569 624298869 47354
261493515 668129504 276999
24991251 832334621 811001
318003297 417180394 755482
763099477 205724176 845093
239849681 233816982 703630
302575811 101521254 581203
832303853 30841845 450080
384792609 19304456 944905
78104599 757949379 285941
255522228 104097155 167567
274659604 356198681 677801
36080087 761731633 382567
280780628 163268223 456201
762131580 142602827 891334
819556210 85178197 884552
665868549 197277149 590027
68762872 674599489 874890
280780628 666628830 768692
550159604 61912804 99079
20245186 371137300 278753
160932458 167350213 404677
238872951 84420898 676163
600172698 316419530 450353
759648607 145085800 802396
609923342 53347265 536411
50066413 149476380 359009
239849681 492860490 977835
654307939 208837759 236421
301040433 369962030 468733
582339638 316238343 942531
630858285 367148406 703113
444496854 288213317 182618
636686676 310722782 165158
78104599 510081098 772346
52047090 198506987 259740
160932458 132173570 156445
505474165 82711532 819603
315836198 541489674 516039
732710171 58919673 746346
389894200 185294848 124717
80623029 203098540 866840
743362361 16286246 958607
80623029 180870486 366383
710051497 33310864 830764
519691766 248722006 371736
306008073 359622747 963030
80623029 573385275 855063
389894200 344236355 241584
68762872 116523335 715296
185286207 451400469 768388
359619383 35657169 134914
385785510 217827362 224718
274552368 379755571 676671
31158977 804895001 797577
391382486 336007436 758478
27480491 887269437 367041
404097065 29464501 766278
552957109 300393707 401909
80623029 118919764 629996
78856903 182636612 584854
27480491 350850340 460550
47117715 680272207 596843
592808611 311925796 461944
444048851 75642915 931805
592808611 115368947 355140
239849681 599136641 969261
31158977 867419004 666898
283721569 189945094 966514
70175164 232400647 81317
819556210 98820414 330691
537528419 384794406 651931
262324311 576662011 751686
665868549 281540909 145234
224251850 232512673 346553
78856903 236979295 793680
68762872 519422825 102997
283266743 639056082 456182
80623029 646766893 350891
315836198 537514618 938187
537528419 55280192 325635
704898917 277349661 377785
292154853 561195963 133199
78104599 510081098 505690
199542793 747866665 16510
293106028 88019272 845868
160932458 312734205 946597
437893642 325205835 634619
24991251 364902949 400900
162560631 412628417 701826
80623029 43975264 857427
36080087 882296537 369271
413660064 187182475 280516
274552368 538587344 840630
575189048 24983650 413603
52047090 787133781 352340
250554077 731694501 967028
293106028 332693059 193685
612072408 185739312 601616
384792609 531799619 467470
592808611 8033928 155318
65113360 639785557 860011
609923342 194632288 269065
250035072 518378700 764189
68762872 506426176 506283
323986346 533339526 256933
27480491 446186172 964271
27480491 764149353 992824
630858285 298764734 319932
759648607 97677265 633609
679275401 53434770 149834
47117715 785186138 370859
552783624 113084925 231559
389894200 524855728 982486
391382486 161574623 240683
323986346 264199351 192681
592808611 86466790 738753
70175164 655801284 85771
600842539 78432862 784263
550159604 418664049 551419
916592228 1784396 360093
238872951 79130346 649491
250035072 338150625 662831
786556679 49497299 168211
194106305 731515702 249248
925622007 72384684 112423
839180871 108228587 251511
665630820 259991187 84690
36080087 42024512 699168
135388551 779361377 299440
249808348 404199956 818972
68762872 66625679 895484
194106305 381082743 17520
437893642 18870881 186569
24991251 298302598 453965
250035072 648542909 139321
378330831 284939776 703419
185286207 332732198 565786
550159604 285894374 526611
385785510 260872940 20869
20245186 285762887 405706
283721569 266438035 897586
743362361 186260658 486410
433561566 435326594 660451
280780628 152780938 716050
280780628 482318849 69570
65113360 571573316 326164
897659548 20717076 526475
385785510 27874554 757306
665630820 232028728 292804
122173734 621188627 73178
315836198 365641645 182475
681477843 121331615 667593
65113360 440360805 445180
444496854 453162694 596351
65113360 134429433 707603
441361042 107305424 787352
612072408 42235531 583983
857325872 111497781 88693
280780628 640640051 95378
274659604 169837250 615446
819556210 42960064 454817
36080087 777059625 73648
78104599 775246217 268398
309637450 49981933 468723
29946031 259852709 927785
404097065 398712393 592049
725976448 106327405 982079
609923342 308453282 765434
759648607 237113787 633828
274659604 6121024 523824
135388551 64154242 624227
519691766 405930241 216699
239849681 599331190 606149
427512818 10380824 640290
430714658 235153891 468965
78856903 795523978 842226
78104599 317171953 617108
519691766 402631059 714967
250554077 747452614 273604
239849681 425781139 743221
391382486 313516431 63809
413660064 123868355 670972
381125300 461658322 139635
444496854 353314866 100940
29172630 546016418 501108
301040433 251916676 161565
552783624 126491777 6441
665630820 255789859 428654
293106028 12902045 234617
550159604 129115797 539567
588185697 269140175 653186
199542793 401299746 910672
519691766 316362212 977462
523636358 319147264 710209
705667688 223955331 933234
80623029 590379434 872673
665868549 256454276 963101
853350816 11539525 335155
705667688 241741770 678734
70175164 772608458 263465
24991251 843896909 841633
384792609 208016002 409678
52047090 210277221 672389
124598293 115251388 826309
274552368 578798448 183068
588185697 380637956 714290
80623029 310759457 991447
385785510 151742909 373203
588185697 12656842 729942
832303853 89116826 938227
385785510 202400187 102881
588185697 276704644 493269
523636358 279173100 615732
250035072 300124532 89926
444048851 260850066 110569
797811720 118780508 809142
292154853 308017845 696675
519691766 401728913 843107
853350816 61399112 605682
135388551 543886850 998457
456764523 525484055 340237
224251850 441378970 878831
160932458 835829936 298423
819556210 177206184 986606
249808348 518605424 285039
404097065 543312393 950937
135388551 417395073 954871
328282671 283789737 845917
29172630 953075948 430571
24991251 408570315 474651
250035072 611147611 148587
836053978 82322646 572183
623830367 305792652 291956
681477843 116333877 713605
552783624 71046743 439978
433561566 237440897 424691
473666663 252309785 926269
305779492 340878958 681069
250035072 460016425 873876
124598293 36334165 752250
121159047 482453825 370817
20245186 272860842 494443
444048851 281927597 421510
904734407 3286031 555234
609923342 253222356 227768
588185697 48500979 114547
609923342 358900311 516637
249808348 512323232 455218
413660064 83505788 77463
274659604 404615797 31704
50066413 354030652 774224
389894200 423245512 367502
853350816 15537344 656845
588185697 310392284 165686
636686676 361320015 535478
550159604 25029444 868721
518018405 345127293 825396
162560631 99763680 542461
315836198 503720012 596451
289798740 91326560 623368
239849681 164247384 685736
85592555 358456296 394407
306008073 212010332 90600
305779492 53839891 767658
249808348 413462259 72427
832303853 32586488 685485
194106305 111901768 303
786556679 121463759 544046
505474165 424148854 252951
20245186 739403421 419458
671002463 191513811 676233
68762872 454873486 434724
293106028 282083020 335690
238872951 580683259 450352
519691766 33091858 854187
224251850 608052003 298616
315836198 207800160 194952
47117715 576712652 377922
224251850 614929021 763462
759648607 165973400 341134
389894200 1488286 331550
122173734 690965978 478529
301040433 4739059 454493
929623019 68383672 259827
743362361 121527980 81478
323293849 68088637 341376
732710171 1420384 215362
600172698 23657669 115083
842783622 116245173 398679
582339638 271011178 830307
238872951 686749056 720823
224251850 358087788 420247
378330831 547291176 905209
384792609 269215695 112860
385785510 522234928 460116
395276552 78390111 101398
832303853 28878830 851102
588185697 254597925 277414
575189048 347133777 649338
625799087 109384604 715544
413660064 443665808 64549
359619383 54040681 868040
80623029 323474036 557762
70175164 185347064 723167
78104599 626794318 165189
395276552 338854003 393347
727389922 75419536 694145
441361042 527462611 444591
473666663 252309785 310042
328282671 569376877 14019
68762872 794382826 765590
31158977 297123694 342518
804555630 164268023 474670
194106305 592450374 222823
52047090 877575929 411133
791629844 40674009 92670
68762872 369130770 282439
121159047 128876025 689041
600842539 260340144 794979
625799087 109384604 800254
523636358 139634249 382971
68762872 735792758 553154
444048851 315599756 702227
582339638 143636810 276224
389894200 449092122 908523
625799087 53476314 270340
160932458 641877000 130892
384792609 320875079 208496
289798740 2356113 973321
70175164 877234294 461522
302575811 554750061 763827
552802426 154683 285873
52047090 682083465 910560
813139712 169108866 632403
623830367 84347191 761563
124598293 554677108 813723
603612872 235567999 921549
274659604 262868815 684207
636686676 202494195 821984
29946031 802357822 124555
65113360 365601298 876697
832303853 115105605 939626
29946031 6134056 787117
303145118 459954359 226737
519691766 377967782 361134
389894200 162908226 667923
305779492 526524361 536189
261493515 659927164 391896
504930328 200737360 579320
323293849 474517871 286011
292154853 561195963 193564
302575811 566312349 849728
504930328 314625882 521213
162560631 441052241 508817
85592555 830999673 679233
121159047 741986651 644019
710051497 187608051 82928
50066413 559856929 634948
303145118 601589289 936306
384792609 168164500 157447
318003297 336005007 586924
504930328 357585946 27793
292154853 31138996 704099
550159604 446602790 249396
444048851 424839309 107833
734130555 262631839 80105
804555630 142853828 120789
504930328 87878283 930582
289798740 228219665 4782
162560631 412628417 669745
456764523 412123637 87819
609923342 94975575 27961
239849681 551780163 297233
194106305 574307467 184244
441361042 327052730 950983
762131580 80652042 821453
261493515 41082296 41043
305779492 537004130 657309
804555630 48795186 982925
922322825 59111965 84249
623830367 250550514 448440
705667688 210924540 751144
518018405 186880512 84847
318003297 5983049 328046
381125300 566284158 846973
437893642 558868752 638214
433561566 116598038 36828
575189048 267594574 719787
797811720 200194971 468127
199542793 92612060 776890
404097065 144569401 146517
68762872 361951786 452386
582339638 260443984 80095
124598293 673213427 240937
623830367 335198428 648927
381125300 123805028 388550
162560631 390241795 300034
665630820 232028728 85686
385785510 456998112 515783
378330831 520247150 945485
29946031 285890167 491094
323293849 411889842 247354
523636358 345251802 340248
433561566 159247045 502773
135388551 417413875 894636
162560631 31545674 879960
523636358 102162729 872921
250554077 403454227 163284
70175164 609100237 788930
735183691 51372988 973875
636686676 166122782 22077
391382486 279619977 54111
29172630 889203994 548231
437893642 144445996 725848
665868549 238865858 918736
305779492 437582869 480884
665868549 248881379 366087
732710171 124615701 465415
31158977 372938088 896441
391382486 530038193 53369
413660064 17054594 703010
315836198 665598592 995708
679275401 64086960 221690
194106305 641947673 236661
309637450 395261467 254485
473666663 448656162 401713
518018405 214691766 993784
743362361 161372046 864192
705667688 202352750 679090
47117715 755691743 357418
122173734 487749608 245049
199542793 662973481 680904
323293849 481261781 539664
262324311 61662035 621579
378330831 482851852 761243
359619383 25173226 960140
289798740 262984884 598691
665868549 203019611 622943
359619383 367770539 562093
303145118 2634374 727027
36080087 225413428 515807
47117715 791868607 529451
293106028 625270596 342974
274552368 630182039 197548
727389922 85749790 836005
122173734 262618875 767986
224251850 220245004 812027
623830367 103559555 619949
262324311 242606017 485962
