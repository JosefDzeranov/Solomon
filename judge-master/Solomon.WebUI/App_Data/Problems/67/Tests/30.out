22823325
31
18 894 679 486 246 791 957 144 481 721 880 991 815 987 559 608 615 826 883 146 760 81 106 78 838 77 961 930 502 2 207 